UIKitDz/BirthdaysList.swift

```diff
@@ -0,0 +1,180 @@
+// BirthdaysList.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Класс с днями рождениями
+class BirthdaysList: UIViewController {
```

Comment:
не забывай писать final
```suggestion
final class BirthdaysList: UIViewController {
```